src/OrchardCore.Modules/OrchardCore.Resources/wwwroot/Scripts/menu.js

```diff
@@ -0,0 +1,28 @@
+/*
+** NOTE: This file is generated by Gulp and should not be edited directly!
+** Any changes made directly to this file will be overwritten next time its asset group is processed by Gulp.
+*/
+
+var activeClass = 'active';
+
+$(document).ready(function () {
+    // Find navbar item with current url and set active.
+    var currentUrl = window.location.href.replace(window.location.protocol + '//' + window.location.host, '');
+    $('li > a[href="' + currentUrl + '"]').parent().addClass(activeClass);
+
+    // Smooth scrolling on click and set active.
+    $('li > a').click(function (e) {
```

Comment:
Super opinionated. This part should be in a theme only, not in the resources folder. And if the theme doesn't already have it, then just remove it.